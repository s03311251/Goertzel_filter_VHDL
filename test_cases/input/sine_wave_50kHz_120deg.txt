3BB6
3569
2D03
2358
1958
1000
0838
02C4
002D
00B3
0449
0A96
12FC
1CA7
26A7
2FFF
37C7
3D3B
3FD2
3F4C
3BB6
3569
2D03
2358
1958
1000
0838
02C4
002D
00B3
0449
0A96
12FC
1CA7
26A7
2FFF
37C7
3D3B
3FD2
3F4C
3BB6
3569
2D03
2358
1958
1000
0838
02C4
002D
00B3
0449
0A96
12FC
1CA7
26A7
2FFF
37C7
3D3B
3FD2
3F4C
3BB6
3569
2D03
2358
1958
1000
0838
02C4
002D
00B3
0449
0A96
12FC
1CA7
26A7
2FFF
37C7
3D3B
3FD2
3F4C
3BB6
3569
2D03
2358
1958
1000
0838
02C4
002D
00B3
0449
0A96
12FC
1CA7
26A7
2FFF
37C7
3D3B
3FD2
3F4C
