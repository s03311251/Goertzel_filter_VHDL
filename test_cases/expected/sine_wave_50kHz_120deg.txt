000A2320
00043AE0
